src/components/organize/people/PersonDetailsCard.tsx

```diff
@@ -1,46 +1,116 @@
-import {
-  Divider,
-  List,
-  ListItem,
-  ListItemIcon,
-  ListItemText,
-} from '@material-ui/core';
-import { LocationCity, Mail, Phone } from '@material-ui/icons';
-
-import PersonCard from './PersonCard';
-import { ZetkinPerson } from 'types/zetkin';
-
-const PersonDetailsCard: React.FunctionComponent<{ person: ZetkinPerson }> = ({
-  person,
-}) => {
-  const details = [
-    { icon: <Phone />, value: person.phone },
-    { icon: <Mail />, value: person.email },
-    {
-      icon: <LocationCity />,
-      value: [person.street_address, person.city]
-        .filter((item) => !!item)
-        .join(', '),
-    },
-  ].filter((detail) => !!detail.value);
+import NextLink from 'next/link';
+import { OpenInNew } from '@material-ui/icons';
+import { useRouter } from 'next/router';
+import { Button, Card, Link, ListItem, ListItemText } from '@material-ui/core';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import ZetkinDate from 'components/ZetkinDate';
+import ZetkinList from 'components/ZetkinList';
+import ZetkinSection from 'components/ZetkinSection';
+import { ZetkinCustomField, ZetkinPerson } from 'types/zetkin';
+
+const PersonDetailLink: React.FunctionComponent<{
+  href: string;
+}> = ({ children, href }) => (
+  <NextLink href={href} passHref>
+    <Link>{children}</Link>
+  </NextLink>
+);
```

Comment:
Is this something that should (eventually) be replaced by a general-purpose `<ZetkinLink>` component?

Reply:
Maybe/probably, that's something we probably want to do when auditing / implementing a design system